physionet-django/events/templates/events/event_notifications.html

```diff
@@ -53,6 +53,26 @@ <h5 class="modal-title">Respond to participation request</h5>
                                     <div class="modal-body author_participation-response_form">
                                         <p>{{ participation_response_form.instance.user.get_full_name }} is requesting
                                             to join Event {{ participation_response_form.instance.event.title }}.</p>
+
+                                        {% if participation_response_form.instance.event.datasets.exists %}
```

Comment:
These changes do not (as far as I can see) relate to "Adding cohost permissions to view/respond to participation requests".

Reply:
This was added to make the cohosts approving the participant request. This was another point added by Benjamin during one of our discussions.